_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -55462,6 +55461,21 @@
 /obj/effect/turf_decal/bot,
 /turf/open/floor/plasteel,
 /area/quartermaster/miningdock)
+"ANx" = (
+/obj/machinery/autolathe{
+	layer = 3
```

Comment:
dirty var.

Reply:
What's that? I edited the map in the map editor and ran map merger as commit hook.

Reply:
Did I remove them properly? I asked in coderbus and they said text editor is ok